config/routes.rb

```diff
@@ -33,5 +33,10 @@
       end
     end
   end
-
+  match '/',
+        controller: 'application',
+        action: 'options',
+        constraints: { method: %w[OPTIONS POST PUT PATCH DELETE] },
+        via: %i[options post put patch delete]
+  match '*path', via: :all, to: redirect('/404.html')
```

Comment:
misschien ook nog even een 404 pagina toevoegen met het u-can-act logo? Volgens mij kunnen we dat ding van hoegekis wel copypasten, en alleen het plaatje aanpassen.

Reply:
Fixed.